libs/storage/tests/gtest/key_value_index_tests.cpp

```diff
@@ -438,27 +438,8 @@ bool LoadSaveVsBulk()
     random_batched_size = test.size();
   }
 
-  return (batched_hash == bulk_hash) && (random_batched_hash == batched_hash) &&
-         (bulk_size == batched_size) && (random_batched_size == bulk_size);
-}
-
-TEST(storage_key_value_index_gtest, Value_consistency)
-{
-
-  EXPECT_TRUE(ValueConsistency());
-  EXPECT_TRUE((LoadSaveValueConsistency<kvi_type, kvi_type>()));
-  EXPECT_TRUE((LoadSaveValueConsistency<kvi_type, cached_kvi_type>()));
-  EXPECT_TRUE((LoadSaveValueConsistency<cached_kvi_type, kvi_type>()));
-  EXPECT_TRUE((LoadSaveValueConsistency<cached_kvi_type, cached_kvi_type>()));
+  ASSERT_TRUE(batched_hash == bulk_hash && random_batched_hash == batched_hash &&
+              bulk_size == batched_size && random_batched_size == bulk_size);
 }
 
-TEST(storage_key_value_index_gtest, Hash_consistency)
-{
-  EXPECT_TRUE(RandomInsertHashConsistency());
```

Comment:
Difficult to see from the diff have we lost this test coverage

Reply:
No, I split it up into three test cases